collector/generatorreceiver/topos/hipster_shop.json

```diff
@@ -98,7 +98,7 @@
             "downstreamCalls" : { },
             "maxLatencyMillis" : 100,
             "tagSets": [
-              {"inherit": ["starter"]}
+              {"weight": 1, "tags": { "inherit": ["starter"]}} 
```

Comment:
I believe this line should be `{"weight": 1, "inherit":  ["starter"]}`, since `inherit` is a field of the `TagSet` struct. Though we actually haven't implemented the "inherit" functionality yet... I will make a ticket for that 😄

Reply:
Fixed!